src/compiler/scala/tools/nsc/transform/Transform.scala

```diff
@@ -13,12 +13,8 @@
 package scala.tools.nsc
 package transform
 
-/** <p>
- *    A base class for transforms.
- *  </p>
- *  <p>
- *    A transform contains a compiler phase which applies a tree transformer.
- *  </p>
+/** A base class for transforms.
+ *  A transform contains a compiler phase which applies a tree transformer.
```

Comment:
```suggestion
 *  A transform contains a compiler phase that applies a tree transformer.
```

Reply:
That's a good one. I know I've read the Quick and Dirty Tip by Grammar Girl within the last year or two, but I just refreshed my memory. This might be a borderline case because: "A transform contains a phase which applies the transformer." A stronger case would be: "A transform contains a phase that is named SomethingTransformPhase." That it applies a transformer is restrictive; maybe "which" is OK because it's saying a transform has a phase like any other subcomponent; BTW, the phase applies the transformer. But I would take this change anyway. :)